src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -268,9 +368,10 @@ def all_mappings(
         Returns:
           An iterator over all matching deployed artifacts.
         """
-        for src in self._src_to_dest.keys():
-            for deployed_src, deployed_dest in self._mappings_for_source(
+        for src, dest in self._artifact_map:
```

Comment:
`self._artifact_map` is of type `_ArtifactPathMap`. Did you mean
```
for src, dest in self._artifact_map.__src_dest_pairs`:
```

Reply:
No, `_ArtifactPathMap` defines `__iter__` for that very reason.

Reply:
Missed that, thank you!